staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -217,19 +218,33 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "code", "rejected"})
 
+	webhookSLO := metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_slo_duration_seconds",
```

Comment:
wdyt about renaming this to `webhook_admission_slo_duration_seconds` to be consistent with the stable duration metric?

Reply:
Done.